src/net/sourceforge/kolmafia/utilities/StringUtilities.java

```diff
@@ -832,6 +840,14 @@ public static final String lookupPrepositions(final String text) {
     return rv == null ? text : rv;
   }
 
+  public static HashMap<String, String> getCopyOfPrepositionsMap() {
```

Comment:
I'm not sure what the point of taking a copy of the map is? Does it get changed in the test somehow?

`new HashMap<>(prepositionsMap)` should avoid the cast.

Reply:
can you avoid the cast by using `new HashMap`?